### ISSUE TITLE
[Accessibility] Closing dialog does not returen keyboard focus to the element that triggered it

### ISSUE THREAD

**Issue by amberhinds** [ProgressPlanner/brand-assets#17]
* [ ] I've read and understood the [contribution guidelines](https://github.com/Emilia-Capital/.github/blob/main/CONTRIBUTING.md).
* [ ] I've searched for any related issues and avoided creating a duplicate issue.

### Please give us a description of what happened
When closing the brand assets dialog focus is lost and not returned to the logo, which is its triggering element.

#### Step-by-step reproduction instructions
1. Go to https://progressplanner.com/
2. Trigger the dialog by right clicking the logo.
3. Press the Escape key to close the dialog.
4. Press the Tab key and observe that focus is lost, no longer in the header.
5. Repeat process by closing the dialog with the close button.

#### Expected results
When a modal is closed, keyboard focus should be returned to the button that triggered it, ensuring that keyboard-only and screen reader users do not lose their place on the page.

Use JavaScript to return keyboard focus to the button that opened the modal when it is closed (should work with both Escape and when using the close button). This is related to #13 - focus should be returned to that button once it has been added to the page.

#### Actual results
Focus is lost completely when the dialog is closed, and hitting Tab multiple times does not reveal a focus outline.

### Technical info

* If relevant, which editor is affected (or editors): 
- [ ] Block Editor
- [ ] Gutenberg Editor
- [ ] Elementor Editor
- [ ] Classic Editor 
- [ ] Other: 

* Which browser is affected (or browsers): 
- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Other: 

#### Used versions
* Device you are using:
* Operating system:
* PHP version:
* WordPress version: 
* WordPress Theme:
* Plugin version: 
* Gutenberg plugin version: 
* Elementor plugin version: 
* Classic Editor plugin version: 
* Relevant plugins in case of a bug:
